package.json

```
@@ -4,23 +4,25 @@
   "private": true,
   "scripts": {
     "serve": "WEBPACK_TARGET=dev vue-cli-service serve",
-    "dev-server": "concurrently \"tsc -w --p src/server\" \"nodemon serverBuild/server.js\"",
+    "dev-server": "concurrently \"tsc -w --project tsconfig.server.json\" \"nodemon serverBuild/server.js\"",
```

the `servers.js` path changed
```suggestion
    "dev-server": "concurrently \"tsc -w --project tsconfig.server.json\" \"nodemon serverBuild/server/server.js\"",
```

I also get a `server is running at port undefined` but I don't know if that's just me